ansible/roles/openshift_template_deployer/meta/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+galaxy_info:
+  author: OpenShift Ops
+  description: Openshift Operations Volume Provisioner Deployer
+  company: Red Hat, Inc
+  license: ASL 2.0
+  min_ansible_version: 1.2
+  platforms:
+  - name: EL
+    versions:
+    - 7
+dependencies:
+- role: tools_roles/lib_yaml_editor
+- role: tools_roles/lib_openshift_3.2
```

Comment:
we might want to switch this over to use the openshift-ansible lib_openshift and lib_yaml_editor.